parquet-protobuf/src/test/java/org/apache/parquet/proto/ProtoSchemaConverterTest.java

```diff
@@ -82,264 +93,447 @@ public void testConvertAllDatatypes() throws Exception {
    * Tests that all protocol buffer datatypes are converted to correct parquet datatypes.
    */
   @Test
-  public void testProto3ConvertAllDatatypes() throws Exception {
-    String expectedSchema =
-      "message TestProto3.SchemaConverterAllDatatypes {\n" +
-        "  optional double optionalDouble = 1;\n" +
-        "  optional float optionalFloat = 2;\n" +
-        "  optional int32 optionalInt32 = 3;\n" +
-        "  optional int64 optionalInt64 = 4;\n" +
-        "  optional int32 optionalUInt32 = 5;\n" +
-        "  optional int64 optionalUInt64 = 6;\n" +
-        "  optional int32 optionalSInt32 = 7;\n" +
-        "  optional int64 optionalSInt64 = 8;\n" +
-        "  optional int32 optionalFixed32 = 9;\n" +
-        "  optional int64 optionalFixed64 = 10;\n" +
-        "  optional int32 optionalSFixed32 = 11;\n" +
-        "  optional int64 optionalSFixed64 = 12;\n" +
-        "  optional boolean optionalBool = 13;\n" +
-        "  optional binary optionalString (UTF8) = 14;\n" +
-        "  optional binary optionalBytes = 15;\n" +
-        "  optional group optionalMessage = 16 {\n" +
-        "    optional int32 someId = 3;\n" +
-        "  }\n" +
-        "  optional binary optionalEnum (ENUM) = 18;" +
-        "  optional int32 someInt32 = 19;" +
-        "  optional binary someString (UTF8) = 20;" +
-        "  optional group optionalMap (MAP) = 21 {\n" +
-        "    repeated group key_value {\n" +
-        "      required int64 key;\n" +
-        "      optional group value {\n" +
-        "        optional int32 someId = 3;\n" +
-        "      }\n" +
-        "    }\n" +
-        "  }\n" +
-        "}";
+  public void testProto3ConvertAllDatatypes() {
+    String expectedSchema = JOINER.join(
```

Comment:
is it possible to separate this tpe of code style cleanup from functional changes?

Reply:
wdym by "tpe"?

if this isn't blocking, i'd rather avoid the busy-work to undo and redo in a different branch.